src/main/java/com/booksaw/betterTeams/Main.java

```diff
@@ -192,6 +209,10 @@ public void onDisable() {
 			HologramManager.holoManager.disable();
 		}
 
+		if (luckPermsManager != null) {
+			luckPermsManager.unregister();
```

Comment:
Will this fail if 
`luckPermsManager.register();` in the setup script fails, as it will not be null, but presumably will not be registered